### PR TITLE
Adding documentation for API DNS tests. Corrections. Images optimization.

### PR DESCRIPTION
**Pull request by cosme-ddog**
### What does this PR do?
- Adding documentation for new Synthetics Monitoring API test for DNS.
- Corrections for other API tests.
- Lossless optimization of existing PNGs using ImageOptim.

### Motivation
New API test: DNS.

### Preview link

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

Check preview base path using the URL in details in `preview` status check.

### CODE REVIEW

File: content/en/synthetics/api_tests/_index.md [hunk 1]
```diff
@@ -93,6 +93,23 @@ Define the request you want to be executed by Datadog:
 [3]: /synthetics/private_locations/
 {{% /tab %}}
 
+{{% tab "DNS Test" %}}
+
+{{< img src="synthetics/api_tests/dns_test.png" alt="Define DNS Request"  style="width:80%;" >}}
+
+1. **Choose request type**: `DNS`
```
Comment (apigirl) [inline content/en/synthetics/api_tests/_index.md:100]:
```suggestion
1. **Choose a request type**: `DNS`.
```

**Review by apigirl — APPROVED** [review]

File: content/en/synthetics/api_tests/_index.md (hunk 1 above)
Comment (cosme-ddog) [reply content/en/synthetics/api_tests/_index.md:100]: I guess we should also change for HTTP, SSL, and TCP tests then?

File: content/en/synthetics/api_tests/_index.md [hunk 2]
```diff
@@ -93,6 +93,23 @@ Define the request you want to be executed by Datadog:
...
+2. Specify the `Domain` and optional `DNS Server` to use.
+3. **Name**: The name of your DNS test.
+4. **Select your tags**: The tags attached to your DNS test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetic Monitoring page][1].
+5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetic API test on a private endpoint not accessible from the public internet.
+6. **How often should Datadog run the test?** Intervals are available between every five minutes to once per week.
```
Comment (MargotLepizzera) [inline content/en/synthetics/api_tests/_index.md:105]:
```suggestion
6. **How often should Datadog run the test?** Intervals are available between every one minute to once per week.
```

File: content/en/synthetics/api_tests/_index.md [hunk 3]
```diff
@@ -61,11 +61,11 @@ Define the request you want to be executed by Datadog:
...
-{{< img src="synthetics/api_tests/make-ssl-request.png" alt="Make SSL Request"  style="width:80%;" >}}
+{{< img src="synthetics/api_tests/make-ssl-request.png" alt="Define SSL Request"  style="width:80%;" >}}
 
 1. **Choose request type**: `SSL`
 2. Specify the `Host` and the SSL `Port`. By default, the port is set to _443_.
-3. **Name**: The name of your API test.
+3. **Name**: The name of your SSL test.
 4. **Select your tags**: The tags attached to your SSL test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetic Monitoring page][1].
 5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetic API test on a private endpoint not accessible from the public internet.
 6. **How often should Datadog run the test?** Intervals are available between every five minutes to once per week.
```
Comment (MargotLepizzera) [inline content/en/synthetics/api_tests/_index.md:71]:
```suggestion
6. **How often should Datadog run the test?** Intervals are available between every one minute to once per week.
```

File: content/en/synthetics/api_tests/_index.md [hunk 4]
```diff
@@ -78,11 +78,11 @@ Define the request you want to be executed by Datadog:
...
-{{< img src="synthetics/api_tests/tcp_test.png" alt="Make TCP Request"  style="width:80%;" >}}
+{{< img src="synthetics/api_tests/tcp_test.png" alt="Define TCP Request"  style="width:80%;" >}}
 
 1. **Choose request type**: `TCP`
 2. Specify the `Host` and the TCP `Port`.
-3. **Name**: The name of your API test.
+3. **Name**: The name of your TCP test.
 4. **Select your tags**: The tags attached to your TCP test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetic Monitoring page][1].
 5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetic API test on a private endpoint not accessible from the public internet.
 6. **How often should Datadog run the test?** Intervals are available between every five minutes to once per week.
```
Comment (MargotLepizzera) [inline content/en/synthetics/api_tests/_index.md:88]:
```suggestion
6. **How often should Datadog run the test?** Intervals are available between every one minute to once per week.
```

File: content/en/synthetics/api_tests/_index.md [hunk 5]
```diff
@@ -61,11 +61,11 @@ Define the request you want to be executed by Datadog:
...
 
-{{< img src="synthetics/api_tests/make-ssl-request.png" alt="Make SSL Request"  style="width:80%;" >}}
+{{< img src="synthetics/api_tests/make-ssl-request.png" alt="Define SSL Request"  style="width:80%;" >}}
 
 1. **Choose request type**: `SSL`
 2. Specify the `Host` and the SSL `Port`. By default, the port is set to _443_.
-3. **Name**: The name of your API test.
+3. **Name**: The name of your SSL test.
 4. **Select your tags**: The tags attached to your SSL test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetic Monitoring page][1].
 5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetic API test on a private endpoint not accessible from the public internet.
```
Comment (MargotLepizzera) [inline content/en/synthetics/api_tests/_index.md:70]:
```suggestion
5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run your Synthetic SSL test on a private host not accessible from the public internet.
```

File: content/en/synthetics/api_tests/_index.md [hunk 6]
```diff
@@ -78,11 +78,11 @@ Define the request you want to be executed by Datadog:
...
 
-{{< img src="synthetics/api_tests/tcp_test.png" alt="Make TCP Request"  style="width:80%;" >}}
+{{< img src="synthetics/api_tests/tcp_test.png" alt="Define TCP Request"  style="width:80%;" >}}
 
 1. **Choose request type**: `TCP`
 2. Specify the `Host` and the TCP `Port`.
-3. **Name**: The name of your API test.
+3. **Name**: The name of your TCP test.
 4. **Select your tags**: The tags attached to your TCP test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetic Monitoring page][1].
 5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetic API test on a private endpoint not accessible from the public internet.
```
Comment (MargotLepizzera) [inline content/en/synthetics/api_tests/_index.md:87]:
```suggestion
5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run your Synthetic TCP test on a private port not accessible from the public internet.
```

File: content/en/synthetics/api_tests/_index.md (hunk 5 above)
Comment (MargotLepizzera) [reply content/en/synthetics/api_tests/_index.md:70]: And I cannot seem to suggest on the HTTP test section but we probably want to do the same change on that one as well

File: content/en/synthetics/api_tests/_index.md [hunk 7]
```diff
@@ -93,6 +93,23 @@ Define the request you want to be executed by Datadog:
...
 
+{{% tab "DNS Test" %}}
+
+{{< img src="synthetics/api_tests/dns_test.png" alt="Define DNS Request"  style="width:80%;" >}}
+
+1. **Choose request type**: `DNS`
+2. Specify the `Domain` and optional `DNS Server` to use.
+3. **Name**: The name of your DNS test.
+4. **Select your tags**: The tags attached to your DNS test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetic Monitoring page][1].
+5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetic API test on a private endpoint not accessible from the public internet.
```
Comment (MargotLepizzera) [inline content/en/synthetics/api_tests/_index.md:104]:
```suggestion
5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetic DNS test on a private domain not accessible from the public internet.
```

File: content/en/synthetics/api_tests/_index.md [hunk 8]
```diff
@@ -143,8 +162,25 @@ If you click on **Test URL**, then the basic assertion is automatically filled:
...
 If you click on **Test URL**, then the basic assertion is automatically filled:
 
-- `response time` _is less than_ 2000 ms.
+- `response time` _is less than_ 1000 ms.
+
+{{% /tab %}}
+
+{{% tab "DNS Test" %}}
 
+| Type                | Subtypes                | Operator                                           | Value type                 |
```
Comment (MargotLepizzera) [inline content/en/synthetics/api_tests/_index.md:171]: What about?
```suggestion
| Type                | Record type                | Operator                                           | Value type                 |
```

File: content/en/synthetics/api_tests/_index.md (hunk 1 above)
Comment (cosme-ddog) [reply content/en/synthetics/api_tests/_index.md]: I changed for all types.

**Pull request merged**